bin/triggers_check

```diff
@@ -4,7 +4,7 @@
 require 'bundler/setup'
 require 'dotenv/load'
 require 'optparse'
```

Comment:
`require 'optparse'` can be removed too?

Reply:
I left it because it's used in this file.